liwords-ui/src/store/store.tsx

```diff
@@ -279,7 +279,7 @@ const gameStateInitializer = (
   onClockTick: (p: PlayerOrder, t: Millis) => void,
   onClockTimeout: (p: PlayerOrder) => void
 ) => {
-  const state = defaultGameState;
+  const state = { ...defaultGameState };
```

Comment:
Might be cleaner as
return {
  ...defaultGameState,
  clockController,
  onClockTick,
  onClockTimeout,
};